README.md

```diff
@@ -1 +1,679 @@
+date: 2022-03-31T17:37:03Z
+description: "Functional programming styles have some advantages: predictability and great debugability. Those rely on well-defined patterns that implicitly avoid that you end up shooting yourself in the foot. The newly proposed javascript pipe operator comes without this protection and has a good chance of introducing hard-to-understand code and bad bugs. Here's why."
+categories: ["functional", "f#", "javascript", "pipe"]
+authors: [ "Matthias Kainer", ]
+toc: true
+dropCap: true
+displayInMenu: false
+displayInList: true
+draft: true
+resources:
+- name: featuredImage
+  src: "js-pipe-op.png"
+  params:
+    description: "A code example of the js pipe operator from the article"
+---
+
+The world of software development evolves at a ludicrous speed. Things considered state of the art 20 years ago when I started are now considered legacy and bad practice. It's as if doctors would have gone from leeches to gene splicing in half a dozen years. Luckily, I enjoy learning. And I appreciate every new ideas and pattern that I get to know with every new language I encounter. The borrow checker and the high level features at high performance from Rust. The easiness of bringing your thoughts about data into code from Python. And the convenience of composing functions with f#, which always makes me feel as if I'm the maestro of the code, timing the different parts of the orchestra ideally to create the most beautiful song. 
+
+And all the things, the big and the small, I love in those languages, I miss in others. When I switch from Rust to another, I miss results and matches. When I switch from Python to another language, I search for their equivalent of NumPy and panda. When I switch from f#, I search for a way to pipe and bind my data.
+
+It seems a few people were coming from f# to javascript and missed that too. Thus, a new feature will be added to the language: The [pipeline operator](https://github.com/tc39/proposal-pipeline-operator). When I first read about this idea, it got me excited. Would I be able to create my workflows in javascript soon? However, after playing around with it, I'm much less enthusiastic and feel that the way this is supposed to be implemented is a mistake. 
+
+Languages have an inherent design that will shape how the language can evolve without contracting itself. Those traits are defined by how the language manages data and state, how it works with types and the path on which it establishes the absence of things—all of those bring together constraints on how one can implement specific patterns in a language. Or, like here, can't.
+
+This very extension looks like a timely topic that will help create more readable code. And on the first look, this is indeed achieved. Rather than a fragment like this `console.log(format(tokenize(readLines(test_data))));`, which we have to read inside-out to understand what it is we want to console.log, we get a statement that we can read almost like a sentence:
```

Comment:
"Helping with an issue people are struggling more, as functional paradigms become more prominent". But maybe I'll just drop the timely rather than blowing up the sentence :D